be/src/exec/pipeline/hashjoin/spillable_hash_join_probe_operator.cpp

```diff
@@ -252,11 +253,10 @@ Status SpillableHashJoinProbeOperator::_load_partition_build_side(workgroup::Yie
     enum SpillLoadPartitionStage { BEGIN = 0, FINISH = 1 };
     ctx.total_yield_point_cnt = FINISH;
     auto wg = ctx.wg;
-    int64_t time_spent_ns = 0;
     while (!finish && !_is_finished) {
-        BREAK_IF_YIELD(wg, yield, time_spent_ns);
+        BREAK_IF_YIELD(wg, &ctx.need_yield, ctx.time_spent_ns);
```

Comment:
ctx.time_spent_ns need be `&`

Reply:
or just input `ctx` is oK ?

Reply:
> ctx.time_spent_ns need be `&`

time_spent_ns is only read in this macro, won't be written, passing by value is ok.

Reply:
> or just input `ctx` is oK ?

yeah, I think some macros can be simplified, but this is not a big problem. I will deal with it later in PR.